### ISSUE TITLE
New-ProviderConfig : The term 'New-ProviderConfig' is not recognized as the name of a cmdlet, function, script file

### ISSUE THREAD

**Issue by mackwan84** [ebekker/ACMESharp#121]
New-ProviderConfig : The term 'New-ProviderConfig' is not recognized as the name of a cmdlet, function, script file

**Comment by bseddon** [comment]
Yes, such a command is not provided by ACMESharp.  What are you trying to do?

**Comment by bseddon** [comment]
You have not provided any information about the commands you have run or the errors you see when you try to run those commands.  You will need to provide this information if you would like help.  The same information contained in the article is available in the [Quick Start](https://github.com/ebekker/ACMESharp/wiki/Quick-Start) guide on the ACMESharp wiki

**Comment by mackwan84** [comment]
Here is my codes. Thanks a lot for your help. 

> Import-Module ACMESharp
> $domain = ""
> $certificiatePassword = ""
> $email = ""
> $vault = "C:\SSL\{0}\{1}" -f $domain, [guid]::NewGuid()
> mkdir $vault
> cd $vault
> 
> Initialize-ACMEVault -BaseURI https://acme-v01.api.letsencrypt.org/
> New-ACMERegistration -Contacts mailto:$email
> Update-ACMERegistration -AcceptTOS
> 
> New-ACMEIdentifier -Dns $domain -Alias dns1
> New-ACMEProviderConfig -WebServerProvider Manual -Alias manualHttpProvider -FilePath $vault\answer.txt
> 
> Get-ACMEIdentifier -Ref dns1
> $completedChallenge = Complete-ACMEChallenge -Ref dns1 -Challenge http-01 -ProviderConfig manualHttpProvider
> $challengeAnswer = ($completedChallenge.Challenges | Where-Object { $_.Type -eq "http-01" }).ChallengeAnswer
> $key = $challengeAnswer.Key
> Write-Host ""
> Write-Host "Create folder structure on $domain like so:"
> Write-Host "$domain/$key"
> Write-Host "Put an index.html file in that location that contains:"
> Write-Host $challengeAnswer.Value
> 
> 
> $challenge = Submit-ACMEChallenge -Ref dns1 -Challenge http-01
> While ($challenge.Status -eq "pending") {
>     Start-Sleep -m 500 # wait half a second before trying
>     Write-Host "Status is still 'pending', waiting for it to change..."
>     $challenge = Update-ACMEIdentifier -Ref dns1
> }
> 
> If($challenge.Status -eq "valid") {
>     New-ACMECertificate -Identifier dns1 -Alias cert1 -Generate
>     $certificateInfo = Submit-ACMECertificate -Ref cert1
> 
>     While([string]::IsNullOrEmpty($certificateInfo.IssuerSerialNumber)) {
>         Start-Sleep -m 500 # wait half a second before trying
>         Write-Host "IssuerSerialNumber is not set yet, waiting for it to be populated..."
>         $certificateInfo = Update-ACMECertificate -Ref cert1
>     }
> 
>     Get-ACMECertificate -Ref cert1 -ExportPkcs12 cert1-all.pfx -CertificatePassword $certificiatePassword
> 
>     Write-Host "All done, there's a cert1-all.pfx file in $vault with password $certificiatePassword for you to use now"
> } Else {
>     $message = "Status is '{0}', can't continue as it is not 'valid'." -f $challenge.Status
>     Write-Host $message
> }

**Comment by bseddon** [comment]
You have provided the command you have used but you have not provided the messages output by ACMESharp. Without these it is not possible to help you.  How do you know it has failed?
